lib/storage/drbd.py

```diff
@@ -398,6 +398,19 @@ def _AssembleLocal(self, minor, backend, meta, size):
       if result.failed:
         base.ThrowError("drbd%d: can't attach local disk: %s",
                         minor, result.output)
+   
+    # syncer init only for drbd => 8.4 - in 8.4 it must be set after local, not in net
+    info = DRBD8.GetProcInfo()
+    version = info.GetVersion()
+    if version["k_minor"] >= 4:
```

Comment:
I think it's best to abstract this check into a separate function and use it in both places. Something like:

```python
@staticmethod
def _NeedsLocalSyncerParams():
  info = DRBD8.GetProcInfo()
  version = info.GetVersion()
  return version["k_minor"] >= 4
```

Then you could write:

```python
def _AssembleLocal(self, minor, backend, meta, size):
  …
  if self._NeedsLocalSyncerParams():
    sync_errors = …
…

def _AssembleNet(self, minor, net_info, dual_pri=False, hmac=None,
                   secret=None):
  …
  if not self._NeedsLocalSyncerParams():
    sync_errors = …
…

This leads to cleaner code and keeps the version check logic in one place.